testchain/runtime/src/lib.rs

```diff
@@ -448,16 +448,13 @@ impl NativeCurrencyKey for SpacewalkNativeCurrencyKey {
 // because this is used in the benchmark_utils::DataCollector when feeding prices
 impl XCMCurrencyConversion for SpacewalkNativeCurrencyKey {
 	fn convert_to_dia_currency_id(token_symbol: u8) -> Option<(Vec<u8>, Vec<u8>)> {
-		// todo: this code results in Execution error:
-		// todo: \"Unable to get required collateral for amount\":
-		// todo: Module(ModuleError { index: 19, error: [0, 0, 0, 0], message: None })", data: None
-		// } cfg_if::cfg_if! {
-		// 	if #[cfg(not(feature = "testing-utils"))] {
-		// 		if token_symbol == 0 {
-		// 			return Some((b"Kusama".to_vec(), b"KSM".to_vec()))
-		// 		}
-		// 	}
-		// }
+		cfg_if::cfg_if! {
```

Comment:
re-enabled, as the only thing required in the vault's integration test, was to specify the `testing-utils` feature in the cargo.toml of the _client/runtime_